recipes-dom0/dom0-image-weston/dom0-image-weston.bb

```diff
@@ -4,6 +4,7 @@ LICENSE = "MIT"
 
 inherit build_yocto
 inherit xt_quirks
+inherit xt_rcar
```

Comment:
not sure, I will move it to .inc file and will contain all common quirks related to R-Car, not only unpacking